initialise/initialise.go

```diff
@@ -58,9 +65,19 @@ func (e *ExternalServiceList) GetConsumer(ctx context.Context, cfg *config.Confi
 }
 
 // GetProducer returns a kafka producer, which might not be initialised yet.
-func (e *ExternalServiceList) GetProducer(ctx context.Context, kafkaBrokers []string, topic string, name KafkaProducerName, envMax int) (kafkaProducer *kafka.Producer, err error) {
+func (e *ExternalServiceList) GetProducer(ctx context.Context, kafkaBrokers []string, topic string, name KafkaProducerName, cfg *config.Config) (kafkaProducer *kafka.Producer, err error) {
+	envMax, err := strconv.ParseInt(cfg.KafkaMaxBytes, 10, 32)
```

Comment:
`10` & `32` are [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)#:~:text=The%20term%20magic%20number%20or,1%20manuals%20of%20the%201960s.).

It's good practice to use named variables instead of numeric literals. A well named variable makes code easier to read - you don't have to go look what 10 means in this context if the name describes the thing it represents.